content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -204,7 +204,7 @@ plugins.
 * hyperv ([driver installation](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperv-driver))
 Note that the IP below is dynamic and can change. It can be retrieved with `minikube ip`.
 * vmware ([driver installation](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#vmware-unified-driver)) (VMware unified driver)
-* none (Runs the Kubernetes components on the host and not in a VM. It is not recommended to run the none driver on personal workstations. Using this driver requires Docker ([docker install](https://docs.docker.com/install/linux/docker-ce/ubuntu/)) and a Linux environment)
+* none (Runs the Kubernetes components on the host and not in a VM. It is not recommended to run the none driver on personal workstations. Using this driver requires Docker ([docker install](https://docs.docker.com/install/linux/docker-ce/ubuntu/)) and a Linux environment. Non docker runtimes are not supported with none driver. Docker is the only supported container runtime for none.)
```

Comment:
The existing text says “Using this driver requires Docker”. This change would state that three times in total.

@RA489 it sounds like you want to reinforce the message. I think maybe there's a different way to word that though, without seeming so repetitive. What do you think?

(If you can clarify intent here I can try to suggest wording).

Reply:
It is for those who are using non docker runtimes so that they are aware about non docker runtimes are not currently supported with none driver. I am ok with the wording you want to suggest to make it better.